tests/driver_at86rf2xx/cmd.c

```diff
@@ -68,13 +68,7 @@ int ifconfig_list(int idx)
         puts("(err)");
         return 1;
     }
-    printf(", Max.Payload: %u", (unsigned)u16_val);
-
-    res = get((netdev_t *)dev, NETOPT_IPV6_IID, array_val, sizeof(array_val));
-    if (res > 0) {
-        printf("\n           IPv6 IID: ");
-        print_addr(array_val, res);
-    }
+    printf(", Max.Payload: %u\n", (unsigned)u16_val);
```

Comment:
the newline `\n` wasn't there before and seems to be added by the following printf ... or is it needed here?

Reply:
Nope, it is not needed.